tests/bootstrap.php

```diff
@@ -22,9 +20,7 @@
     require_once $file;
 }
 
-/*
- * try to get Symfony's PHPUnit Bridge
- */
+// try to get Symfony's PHPUnit Bridge
```

Comment:
This will be reverted by dev-kit

Reply:
Oh, you're right :(

Reply:
But what about adding this rule for all repos to the dev kit ?

Reply:
Feel free to provide a PR